common/components/CommentsCard/comment.tsx

```diff
@@ -1,24 +1,72 @@
-import { List, Avatar, Image } from 'antd'
+import { Comment as CommentAntd, Tooltip } from 'antd'
 import { useGetUserByIdQuery } from 'common/api/userApi/user.api'
-import { UserOutlined } from '@ant-design/icons'
+import {
+  DislikeFilled,
+  DislikeOutlined,
+  LikeFilled,
+  LikeOutlined,
+} from '@ant-design/icons'
 import { IComment } from 'common/modules/models/Task'
+import { useSession } from 'next-auth/react'
+import { useGetUserByEmailQuery } from 'common/api/userApi/user.api'
+import classNames from 'classnames'
+import moment from 'moment'
 import s from './style.module.scss'
 
-const Comment: React.FC<{ comment: IComment }> = ({ comment }) => {
+const Comment: React.FC<{ comment: IComment; taskId: string }> = ({
+  comment,
+  taskId,
+}) => {
+  const session = useSession()
+  const { data: sessionUser } = useGetUserByEmailQuery(
+    session?.data?.user?.email
+  )
+
   const { data } = useGetUserByIdQuery(comment.id)
   const user = data?.data
 
+  const like = () => {
+    // TODO: Back-End to patch likes (if needed)
+  }
+
+  const dislike = () => {
+    // TODO: Back-End to patch dislikes (if needed)
+  }
```

Comment:
Remove it, please.
We decide to don't implement this feature because It's a common chat between customers and workers.